src/frontend/src/components/organisations/search/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { Button, Input, Label } from 'nhsuk-react-components';
+
+const OrganisationsSearch: React.FC = () => {
+
+    const [name, setName] = useState('');
+    const [postCode, setPostCode] = useState('');
+    const [organisation, setOrganisation] = useState('');
+
+    const handleSearch = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.currentTarget.blur();
+    }
+
+    return (
+        <div data-testid='organisation-search'>
+            <Label isPageHeading={true} size='l'>Organisation search</Label>
+            <div className='nhsuk-u-margin-bottom-3' data-testid='search-by'>Search by <strong>either</strong> name, postcode or managing organisation.</div>
+
+            <Input
+                label="Name"
+                value={name}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => setName(e.target.value)}
+                aria-label="Name"
+                data-testid="name-input"
+                width={20}
+            />
+
+            <div className='nhsuk-u-margin-bottom-2'>Or</div>
+
+            <Input
+                label="Postcode"
+                value={postCode}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => setPostCode(e.target.value)}
+                aria-label="Postcode"
+                data-testid="postcode-input"
+                width={10}
+            />
+
+            <div className='nhsuk-u-margin-bottom-2'>Or</div>
+
+            <Input
+                label="Managing organisation"
+                value={organisation}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => setOrganisation(e.target.value)}
+                aria-label="Managing organisation"
```

Comment:
minor suggestion: since `aria-label` replaces the label text for screen readers and can override other label methods. can we use `aria-labelledby` instead.

Reply:
As discussed it wont work as we are using the NHS react components.